src/lib/config-db.ts

```diff
@@ -62,7 +72,8 @@ export async function setConfig (config: ConfigDb, logger: ComponentLogger): Pro
     await configDb.put('enableWss', config.enableWss)
     await configDb.put('enableWebTransport', config.enableWebTransport)
     await configDb.put('enableGatewayProviders', config.enableGatewayProviders)
-    await configDb.put('debug', config.debug ?? defaultDebug)
+    await configDb.put('debug', config.debug ?? defaultDebug())
+    // ignore private/app-only fields
```

Comment:
private/app-only fields should only be set by their respective conditions in another location. currently this PR is adding the only private field, but there may be more in the future.